tabs-tabs-tabs/tabs.js

```diff
@@ -0,0 +1,42 @@
+document.addEventListener("click", function(e) {
+  function callOnActiveTab(callback) {
+    chrome.tabs.query({}, function(tabs) {
+      for (let tab of tabs) {
+        if (tab.active) {
+          callback(tab);
+        }
+      }
+    });
+  }
+
+  if (e.target.id === "tabs-move-beginning") {
+    callOnActiveTab((tab) => {
+      chrome.tabs.move([tab.id], {index: 0});
+    });
+  }
+
+  if (e.target.id === "tabs-move-end") {
+    callOnActiveTab((tab) => {
+      chrome.tabs.move([tab.id], {index: -1});
+    });
+  }
+
+  else if (e.target.id === "tabs-duplicate") {
+    callOnActiveTab((tab) => {
+      chrome.tabs.duplicate(tab.id);
```

Comment:
This doesn't work in Firefox, because it's not implemented yet. I think we should either (1) not include stuff that's not implemented in Firefox or (2) test for such stuff and log a helpful error.

I think I'd vote for (1). We can add it back when support for duplicate() lands.